build.gradle

```diff
@@ -5,7 +5,7 @@ plugins {
 }
 
 group 'com.bisnode.opa'
-version '0.1.1'
+version '0.2.0-SNAPSHOT'
```

Comment:
Will travis himself remove `SNAPSHOT` element during merge? Or you have to do this manually?

Reply:
You'll need to do so manually when you're ready for release. Note though that the build is not done by Travis but Github Actions such as our other builds here.